wagtail/embeds/embeds.py

```diff
@@ -20,7 +20,12 @@ def get_finder_for_embed(url, max_width=None, max_height=None):
     raise EmbedUnsupportedProviderException
 
 
-def get_embed(url, max_width=None, max_height=None, finder=get_finder_for_embed):
+def get_embed(url, max_width=None, max_height=None):
+    """
+    Retrieve an embed for the given URL using the configured finders.
+
+    Note: The 'finder' parameter has been removed (see GitHub issue #7287).
```

Comment:
```suggestion
```

No need to add this reference, we can see the commits in the history if needed, especially as this is just an 'internal' clean up.